src/storage/btree.c

```diff
@@ -25038,6 +25417,7 @@ btree_range_scan_descending_fix_prev_leaf (THREAD_ENTRY * thread_p, BTREE_SCAN *
       pgbuf_unfix_and_init (thread_p, bts->C_page);
       bts->C_page = prev_leaf;
       VPID_COPY (&bts->C_vpid, &prev_leaf_vpid);
+      COMMON_PREFIX_PAGE_SIZE_RESET (bts);
```

Comment:
The order below seems more appropriate for reviewing code.
```suggestion
      pgbuf_unfix_and_init (thread_p, bts->C_page);
      COMMON_PREFIX_PAGE_SIZE_RESET (bts);
      bts->C_page = prev_leaf;
      VPID_COPY (&bts->C_vpid, &prev_leaf_vpid);
```

Reply:
The purpose is to express my opinion, so if you read it, please resolve it.